File: sommeDisplay/app/src/main/java/com/example/danieljoaquimpaulino/sommedisplay/MainActivity.java
package com.example.danieljoaquimpaulino.sommedisplay;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    EditText input_n1, input_n2;

    TextView display_result;

    Button btn_Calculer;



    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        input_n1= (EditText) findViewById(R.id.Edit1);
        input_n2= (EditText) findViewById(R.id.Edit2);

        display_result= (TextView) findViewById(R.id.display);

        btn_Calculer = (Button) findViewById(R.id.Btn1);

        btn_Calculer.setOnClickListener(new View.OnClickListener()
        {
            @Override
            public void onClick(View v) {

                int n1 = Integer.parseInt(input_n1.getText().toString());
                int n2 = Integer.parseInt(input_n2.getText().toString());

                int Reslt = n1*n2;

                display_result.setText(Integer.toString(Reslt));


            }
        });




    }
}

File: Calculator/app/src/main/java/com/example/danieljoaquimpaulino/calculatrice/MainActivity.kt
package com.example.danieljoaquimpaulino.calculatrice

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
    fun buNumberEvent(view: View ){

        var tvNumber:String= tvShowNumber.text.toString()

        var buSelect = view as Button

        when (buSelect.id){

            bu0.id->{


                tvNumber += "0"
            }

            bu1.id->{
                tvNumber += "1"


            }
            bu2.id->{
                tvNumber += "2"


            }
            bu3.id->{
                tvNumber += "3"


            }
            bu4.id->{
                tvNumber += "4"


            }
            bu5.id->{
                tvNumber += "5"


            }

            bu6.id->{
                tvNumber += "6"


            }
            bu7.id->{
                tvNumber += "7"


            }
            bu8.id->{
                tvNumber += "8"


            }
            bu9.id->{
                tvNumber += "9"


            }
            buPoint.id->{
                tvNumber += "."


            }
            buPlus.id->{
                tvNumber = "-" + tvNumber


            }



        }

      tvShowNumber.text = tvNumber


    }
}

